docusaurus.config.js

```diff
@@ -59,6 +59,19 @@ const config = {
       }),
     ],
   ],
+  plugins: [
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        redirects: [
+          {
+            to: 'https://docs.bpftrace.org/latest',
+            from: '/docs',
```

Comment:
Possible to wildcard? ie `/docs/*`. Previous links were like this: https://bpftrace.org/docs/master.html

Reply:
Ah yeah, I'll fix.